sql/item_pfsfunc.cc

```diff
@@ -0,0 +1,136 @@
+/* Copyright (c) 2000, 2023, Oracle and/or its affiliates.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License, version 2.0,
+   as published by the Free Software Foundation.
+
+   This program is also distributed with certain software (including
+   but not limited to OpenSSL) that is licensed under separate terms,
+   as designated in a particular file or component or in included license
+   documentation.  The authors of MySQL hereby grant you an additional
+   permission to link the program and your derivative works with the
+   separately licensed software that they have included with MySQL.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License, version 2.0, for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA */
+
+/**
+  @file
+
+  @brief
+  This file defines all performance schema native functions
+    FORMAT_PICO_TIME()
+*/
+
+#ifdef USE_PRAGMA_IMPLEMENTATION
+#pragma implementation				// gcc: Class implementation
+#endif
+
+#include "mariadb.h"                          // HAVE_*
+
+/*
+  It is necessary to include set_var.h instead of item.h because there
+  are dependencies on include order for set_var.h and item.h. This
+  will be resolved later.
+*/
+#include "set_var.h"
+
+/** format_pico_time() */
+
+bool Item_func_pfs_format_pico_time::fix_length_and_dec(THD *)
+{
+  set_maybe_null();
+  /* Format is 'AAAA.BB UUU' = 11 characters or 'AAA ps' = 6 characters. */
+  m_value.set_charset(&my_charset_utf8mb3_general_ci);
+  return false;
+}
+
+String *Item_func_pfs_format_pico_time::val_str(String *str __attribute__ ((__unused__)))
+{
+  /* Evaluate the argument */
+  double time_val = args[0]->val_real();
+
+  /* If argument is null, return null. */
+  null_value = args[0]->null_value;
+  if (null_value)
+  {
+    return 0;
+  }
+
+  constexpr uint64_t nano{1000};
```

Comment:
I'm ok to have  initializer list and expression list together, but is missing as reference in Coding standard.
Can you please create new PR to update Coding standard regarding adding expression and initializer list as well as update function declaration with reference that functions should be separated with 2 new lines ?

Reply:
Great. I will add this and push the PR today
Thanks a lot for the review :)